src/viewport.js

```diff
@@ -801,10 +801,10 @@ $.Viewport.prototype = {
      * Returns bounds taking constraints into account
      * Added to improve constrained panning
      * @param {Boolean} current - Pass true for the current location; defaults to false (target location).
-     * @returns {OpenSeadragon.Rect} The bounds after applying constraints. The returned $.Rect contains additonal
-     *                               properties constraintsApplied, xConstrained and yConstrained. These flags indicate
-     *                               whether the viewport bounds were modified by the constraints of the viewer rectangle,
-     *                               and in which dimension(s).
+     * @returns {OpenSeadragon.Rect} The bounds in viewport coordinates after applying constraints. The returned $.Rect
+     *                               contains additonal properties constraintsApplied, xConstrained and yConstrained.
```

Comment:
Typo: `additonal` should be `additional`. Sorry I didn't catch this before!

Reply:
@iangilman fixed.